content/03-reference/01-tools-and-interfaces/02-prisma-client/030-crud.mdx

```diff
@@ -498,6 +502,8 @@ You can also perform nested writes during an update, such as:
 
 The `upsert` query updates an existing or creates a new database record. You can use the `select` and `include` options to determine which properties should be included on the returned object.
 
+To perform arithmeic operations (add, subtract, multiply), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```

Comment:
```suggestion
To perform arithmeic operations (add, subtract, multiply, divide), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```